File: database/supabaseClient.js
// supabaseClient.js
import { createClient } from '@supabase/supabase-js'

// Replace these with your Supabase project's URL and anon key
const supabaseUrl = SUPABASE_URL
const supabaseKey = SUPABASE_KEY
const supabase = createClient(supabaseUrl, supabaseKey)

export default supabase

File: backend/project/supabaseClient.py
import os
from django.conf import settings
# from supabaseClient import create_client

# SUPABASE_URL = os.getenv("SUPABASE_URL")
# SUPABASE_KEY = os.getenv("SUPABASE_KEY")

# supabase = create_client(SUPABASE_URL, SUPABASE_KEY)

def create_client():
    SUPABASE_URL = settings.SUPABASE_URL
    SUPABASE_KEY = settings.SUPABASE_KEY
    return create_client(SUPABASE_URL, SUPABASE_KEY)
